plugins/colorschemes/modus.nix

```diff
@@ -0,0 +1,95 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "modus";
+  isColorscheme = true;
+  originalName = "modus-themes-nvim";
+  defaultPackage = pkgs.vimPlugins.modus-themes-nvim;
+
+  maintainers = [ helpers.maintainers.nwjsmith ];
+
+  settingsOptions = {
+    style =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "modus_operandi"
+          "modus_vivendi"
+        ]
+        ''
+          The theme comes in two styles, `modus_operandi`, a dark variant `modus_vivendi`.
```

Comment:
It feels a bit redundant to list the available styles here when the mkEnum function will do that anyway...

Maybe this and `variant`'s description should instead try to describe what the option actually does and/or what things it affects?

Doesn't look like there's anything to quote in upstream's readme though, so I appreciate if you don't want to do this.

Reply:
Good call. I've cribbed new descriptions [from documentation on the original Emacs themes](https://protesilaos.com/emacs/modus-themes#h:f0f3dbcb-602d-40cf-b918-8f929c441baf).